internal/state/dataplane/configuration.go

```diff
@@ -472,16 +473,24 @@ func getPath(path *v1beta1.HTTPPathMatch) string {
 
 func createFilters(filters []v1beta1.HTTPRouteFilter) Filters {
 	var result Filters
+	requestRedirectSet, requestHeaderModifierSet := false, false
```

Comment:
Personal preference, but I think `var requestRedirectSet, requestHeaderModifierSet bool` is a cleaner way to define these.